server/src/test/java/com/decathlon/ara/cartography/AraExporterTest.java

```diff
@@ -0,0 +1,82 @@
+package com.decathlon.ara.cartography;
+
+import com.decathlon.ara.domain.enumeration.FunctionalityType;
+import com.decathlon.ara.service.dto.functionality.FunctionalityDTO;
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AraExporterTest {
+
+    @InjectMocks
+    private AraExporter sut;
+
+    @Test
+    public void getName_should_return_ARA() {
+        // Given
+        final String expected = "ARA";
+        // When
+        String actualName = this.sut.getName();
+        // Then
+        Assertions.assertThat(actualName).isEqualTo(expected);
```

Comment:
Prefer static imports

Reply:
I don't like static imports since they populate the PermGen space until the JVM exits. In this case it will populate the PermGen until our Maven process finishes.

By using standard imports, when the UT is finished, the import is unloaded :)